typescript/cli/src/status/message.ts

```diff
@@ -12,17 +14,19 @@ export async function checkMessageStatus({
   destination,
   origin,
   selfRelay,
+  dispatchTx,
 }: {
   context: CommandContext;
+  dispatchTx?: string;
   messageId?: string;
   destination?: ChainName;
```

Comment:
Is this arg still used for any logic or just that bit of logging below? If not, recommend removing

Reply:
We're using it for core.getDispatchTx too